### PR TITLE
Add OWNERS file to openstack-helm

### LINKED ISSUE #145: ci: repo templates for issues, automatic labeling, etc.
We need to define contribution guidelines, create issue templates and automatically label based on key words, etc.

### PR DESCRIPTION
**Pull request by wilkers-steve**
Added OWNERS file to the repository to mirror other repos

**What is the purpose of this pull request?**:  To add the OWNERS file to declare reviewers and approvers for openstack-helm

**What issue does this pull request address?**: Fixes #145 

**Notes for reviewers to consider**: N/A

**Specific reviewers for pull request**: @v1k0d3n

---
This change is [Reviewable](https://reviewable.io/reviews/att-comdev/openstack-helm/162)

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+reviewers:
```
Comment (v1k0d3n) [inline OWNERS]: I think we need owners at the top, like the workflow here: https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md

This file will be used in CI gating.

**Comment by v1k0d3n** [comment]
Actually in comparing to other kubernetes *proper* repos (meaning, not the one example I was using as my reference), i think these options are perfectly fine. going to merge in.

**Comment by v1k0d3n** [comment]
@alanmeadows would  like your input just before the final merge (if you give me a lgtm).

**Comment by larryrensing** [comment]
LGTM

**Pull request merged**